src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileReaderService;
+import core.basesyntax.service.CsvFileWriterService;
+import core.basesyntax.service.FruitTransactionParser;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.service.impl.CsvFileWriterServiceImpl;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.TransactionProcessorImpl;
+import java.util.List;
+
+public class Main {
+    private static final String LOCATION_OF_THE_INPUT_FILE = "src/main/resources/file.csv";
+    private static final String LOCATION_OF_THE_REPORT_FILE = "src/main/resources/report.csv";
+    private static CsvFileReaderService csvFileReaderService = new CsvFileReaderServiceImpl();
+    private static FruitTransactionParser fruitTransactionParser = new FruitTransactionParserImpl();
+    private static TransactionProcessor transactionProcessor = new TransactionProcessorImpl();
+    private static ReportGenerator reportGenerator = new ReportGeneratorImpl();
+    private static CsvFileWriterService csvFileWriterService = new CsvFileWriterServiceImpl();

```

Comment:
```suggestion
    private static CsvFileReaderService csvFileReaderService = new CsvFileReaderServiceImpl();
    private static FruitTransactionParser fruitTransactionParser = new FruitTransactionParserImpl();
    private static TransactionProcessor transactionProcessor = new TransactionProcessorImpl();
    private static ReportGenerator reportGenerator = new ReportGeneratorImpl();
    private static CsvFileWriterService csvFileWriterService = new CsvFileWriterServiceImpl();

```
you can make this variables local